bin/checkver.ps1

```diff
@@ -132,7 +133,35 @@ $Queue | ForEach-Object {
         if ($json.checkver.PSObject.Properties.Count -eq 1) { $useGithubAPI = $true }
     }
 
-    if ($json.checkver.re) {
+    if ($json.checkver -eq 'sourceforge' -or $json.homepage.Contains('sourceforge.net') -or $json.checkver.sourceforge.path) {
+        if ($json.homepage -match '\/\/sourceforge\.net\/projects\/(?<project>([\w-]+$|[\w-]+))(?:[/]?)' -or $json.homepage -match '\/\/(?<project>[\w-]+)\.sourceforge\.net') {
+            $url = 'https://sourceforge.net/projects/' + $matches['project'] + '/rss'
+            $regex = '\/\/sourceforge\.net\/projects\/' + $matches['project'] + $sourceforgeRegex
+            if ($json.checkver -ne 'sourceforge' -and $json.checkver.GetType() -eq [System.String]) {
+                $regex = $json.checkver
+            }
+        }
+        else {
+            $url = 'https://sourceforge.net/projects/' + (strip_ext $name) + '/rss'
+            $regex = '\/\/sourceforge\.net\/projects\/' + (strip_ext $name) + $sourceforgeRegex
+        }
+    }
+
+    if ($json.checkver.sourceforge -and $json.checkver.sourceforge.GetType() -eq [System.String]) {
+        $url = 'https://sourceforge.net/projects/' + $json.checkver.sourceforge + '/rss'
+        $regex = '\/\/sourceforge\.net\/projects\/' + $json.checkver.sourceforge + $sourceforgeRegex
+    }
+
+    if ($json.checkver.sourceforge.project) {
+        $url = 'https://sourceforge.net/projects/' + $json.checkver.sourceforge.project + '/rss'
+        $regex = '\/\/sourceforge\.net\/projects\/' + $json.checkver.sourceforge.project + $sourceforgeRegex
+    }
+
+    if ($json.checkver.sourceforge.path) {
+        $url = $url + '?path=' + $json.checkver.sourceforge.path
+    }
+
+    if($json.checkver.re) {
```

Comment:
```suggestion
    if ($json.checkver.re) {
```

(format) space

Reply:
Oh, yes. Bad picking in the merge. Thanks for finding it! Fix coming right up!